### ISSUE TITLE
How to specify maintainers in v2 export CL args

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/augur#327]
Currently (as of a2d4afa16868bc22fc1279dbd7a102f9851bf37e) supplying maintainer information via command line args overwrites any provided auspice config files.

Current CL behaviour:
```
augur export v2 --maintainers "person one" "person two" [--maintainer-urls "url1" "url2"]
```
If you want to do this in snakemake you have to tell it to quote arguments (`:q`)
```
    params:
        maintainers = ["Person One", "Person Two"]
    shell:
        augur export v2 --maintainers {params.maintainers:q} ...
```

[Proposed behaviour](https://github.com/nextstrain/augur/pull/325#issuecomment-514047687) via @huddlej:

```
augur export v2 \
    --maintainers "Emma Hodcroft" \
    --maintainers "Robert Dyrdak" \
  ...
```

Re: URLs. One way to do this via the multiple `--maintainers` arg is to interpret 2 fields as being `[name, url]`, which nicely parallels the config JSON, but may result in `--maintainers James Hadfield` which won't do what you think! Alternative is multiple `--maintainer-urls` but this seems to obfuscate the fact that the two args will be zipped together.

**Comment by emmahodcroft** [comment]
The biggest upside to leaving them as they are currently is that it's really easy to match up name with URL. I think with other options this will be more difficult. 

All options (as far as I can tell) are going to involve finicky, fiddly details (which will also differ between CL and snakemake) that users are very likely to mess up, so I would vote that we pick the one that we can explain in the clearest and most easily understandable manner. 

In the same vein, we could put a link in the `--help` which takes users to a webpage where this is explained in detail with examples.

**Comment by tsibley** [comment]
One thought is that we could borrow the syntax from email addresses (`name <address>`) to specify both maintainer name and URL in a single argument, e.g.:

```
augur export v2 \
    --maintainer "Emma Hodcroft <http://emmahodcroft.com>" \
    --maintainer "Robert Dyrdak <mailto:robert@example.com>" \
  ...
```

I think this usage will be pretty clear to explain in our docs.

It's long been conventional to surround URLs with angle brackets in plain text, which is why Markdown and rST have that syntax.

**Comment by trvrb** [comment]
I like this suggestion by @tsibley for how to treat URLs alongside names. I think I prefer:
```
augur export v2 \
    --maintainers "Emma Hodcroft <http://emmahodcroft.com>" \
        "Robert Dyrdak <mailto:robert@example.com>" \
  ...
```

**Comment by emmahodcroft** [comment]
I like @trvrb's suggestion above, with @tsibley's idea, but with one flag. This also makes it easier to integrate into snakemake, I imagine, since you can continue to have one `param` (or whatever) defined with all this info, rather than having multiple to go to multiple flags.

**Comment by tsibley** [comment]
It doesn't have to be one or the other. argparse supports the same flag being used as both `--maintainers 1 2 3` and `--maintainers 1 --maintainers 2 --maintainers 3`. :-)

**Comment by emmahodcroft** [comment]
This is now implemented in PR #368 . Please have a look (and a test, if you can) and see if you are happy, and we can merge.
